tests/structured_conf/test_structured_config.py

```diff
@@ -1170,3 +1170,32 @@ def test_merge_missing_list_promotes_target_type(self, module: Any) -> None:
         c3 = OmegaConf.merge(c1, c2)
         with raises(ValidationError):
             c3.missing.append("xx")
+
+
+@mark.parametrize(
+    "key, parent_expected, child_expected",
+    [
+        param("int1", MISSING, MISSING, id="int1_inherit_unspecified"),
+        param("int2", MISSING, 5, id="int2_overwrite_unspecified"),
+        param("int3", MISSING, 10, id="int3_overwrite_native_missing"),
+        param("int4", MISSING, 15, id="int4_overwrite_missing"),
+        param("list1", MISSING, [1, 2, 3], id="list1_overwrite_missing"),
+        param("list2", [5, 6], [5, 6], id="list2_inherit_default"),
+        param("dict", MISSING, {"a": 5, "b": 6}, id="dict_overwrite_missing"),
+    ],
+)
+def test_structured_config_inheritance(
```

Comment:
nit: it feels more like one test case rather than seven- since there's no difference in how the test is set up. Is this how unit tests in OC usually structured?

Reply:
You're right -- I felt a little bit funny when I was writing that test because of the inconsistency :P

Updated in https://github.com/omry/omegaconf/pull/826/commits/284b4d8df90e74e1a441292631b9b38b5f38e716.

Reply:
awesome! thanks!